src/extensions/extension-loader.ts

```diff
@@ -62,17 +64,20 @@ export class ExtensionLoader {
   protected autoloadExtensions(getLensRuntimeEnv: () => LensExtensionRuntimeEnv, callback: (instance: LensExtension) => void) {
     return reaction(() => this.extensions.toJS(), (installedExtensions) => {
       for(const [id, ext] of installedExtensions) {
-        let instance = this.instances.get(ext.manifestPath)
+        let instance = this.instances.get(ext.id)
         if (!instance) {
           const extensionModule = this.requireExtension(ext)
           if (!extensionModule) {
             continue
           }
           const LensExtensionClass = extensionModule.default;
           instance = new LensExtensionClass({ ...ext.manifest, manifestPath: ext.manifestPath, id: ext.manifestPath }, ext.manifest);
-          instance.enable(getLensRuntimeEnv())
-          callback(instance)
-          this.instances.set(ext.id, instance)
+          try {
+            instance.enable(getLensRuntimeEnv())
```

Comment:
There is no `getLensRuntimeEnv` anymore in base branch, so please merge with `extensions-api`

Reply:
Merged.